tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es2019",
-    "isolatedModules": true,
+    "isolatedModules": false,
```

Comment:
Why did you need to change the `isolatedModules` to `false` ?

Reply:
I use JSX in a hook file I am creating and it didn't like that it was not an isolated module.. I disabled the option so I could continue to implement/test the feature but I will change it back when I move this MR out of Draft.

Reply:
FWIW exporting something should make it an "isolated module"

Reply:
I tried doing `export {}` as a temporary workaround but it didn't fix it... maybe I needed to restart my development server for it to take into effect.